rear_rider_device/rear_rider_bluetooth_server/src/services/hello_world.py

```diff
@@ -1,4 +1,23 @@
 from bluez.example_gatt_server import GATT_CHRC_IFACE, Characteristic, Service, GObject, dbus
+import subprocess
+
+WLAN_INTERFACE = 'wlan'
+
+class WifiAccessPoint():
+    def __init__(self):
+        pass
+    
+    def turn_off(self):
+        ret = subprocess.call(['sudo', 'rfkill', 'block', WLAN_INTERFACE])
+        return ret == 0
+    
+    def turn_on(self):
+        ret = subprocess.call(['sudo', 'rfkill', 'unblock', WLAN_INTERFACE])
+        return ret == 0
+    
+    def is_on(self):
+        ret = subprocess.run(['sudo', 'rfkill'], stdout=subprocess.PIPE).stdout.decode('utf-8').split()
+        return ret[8] == 'unblocked'
```

Comment:
`ret[8] == 'unblocked'` may not be the most reliable way to test if the wifi is turned on, but it should be good for now hopefully.